spec/integ/crypto/cross-signing.spec.ts

```diff
@@ -105,6 +143,86 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("cross-signing (%s)", (backend: s
                 `[${TEST_USER_ID}].[${TEST_DEVICE_ID}].signatures.[${TEST_USER_ID}].[${sskId}]`,
             );
         });
+
+        newBackendOnly("get cross signing keys from secret storage and import them", async () => {
+            // Return public cross signing keys
+            fetchMock.postOnce("path:/_matrix/client/v3/keys/query", SIGNED_CROSS_SIGNING_KEYS_DATA);
```

Comment:
Although this is fine for now, it's not our normal pattern and I think it might make the test harder to maintain. Better to use an `E2EKeyResponder`  (and call `addCrossSigningData` to tell it about the cross-signing keys data)